src/System.Drawing.Common/src/System/Drawing/BufferedGraphics.cs

```diff
@@ -0,0 +1,52 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+
+namespace System.Drawing
+{
+    /// <summary>
+    /// The BufferedGraphics class can be thought of as a "Token" or "Reference" to the buffer that a
+    /// BufferedGraphicsContext creates. While a BufferedGraphics is outstanding, the memory associated with the
+    /// buffer is locked. The general design is such that under normal conditions a single BufferedGraphics will be in
+    /// use at one time for a given BufferedGraphicsContext.
+    /// </summary>
+    public sealed partial class BufferedGraphics : IDisposable
+    {
+        private Graphics _targetGraphics = null;
+        private readonly IntPtr _targetDC = IntPtr.Zero;
```

Comment:
`null` and `IntPtr.Zero` are the default values for `Graphics` and `IntPtr`... they shouldn't need to be specified.